hca/submission.py

```diff
@@ -3,15 +3,36 @@
 from json_converter.json_mapper import JsonMapper
```

Comment:
the changes in this file all have to do with accessions. Perhaps it can be moved to a class dealing with accession specific logic?